components/partials/CtaWebinar.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+const props = defineProps({
+	date: {
+		type: String,
+		required: true
+	}
+});
+
+//Adjust if two parallel landings running
+const landingLink = computed(() => {
+	if (props.date == '30.05.24') {
```

Comment:
[MInor]
I'd consider introducing constant variables for the date and URLs with descriptive names.